recipe/meta.yaml

```diff
@@ -8,10 +8,14 @@ source:
 
 build:
     number: 1
-    skip: True  # [win]
+    features:
+        - vc9 [win and py27]
+        - vc10 [win and py34]
+        - vc14 [win and py35]
```

Comment:
> Are we enforcing the comment syntax for the pre-processor selectors?

They will fail the linting, though currently linting doesn't take place in the feedstock. That will eventually change though.